src/modules/github/components/pull-list-item/pull-list-item.tsx

```diff
@@ -90,6 +97,15 @@ export const PullListItem = ({
   pull,
   handlePullCheckChange
 }: IPullListItemProps) => {
+  const { translate } = useDictionary()
+  const [dialogData, setDialogData] = useState<
+    | {
+        open: true
+      }
+    | {
+        open: false
+      }
+  >({ open: false })
```

Comment:
Question.
Is it better to use dialog data state for AS? Or is it better to use it as an open state?